rct229/rules/section6/section6rule3.py

```diff
@@ -10,6 +10,8 @@
 from rct229.utils.std_comparisons import std_equal
 
 OFFICE_OPEN_PLAN = schema_enums["LightingSpaceType2019ASHRAE901TG37"].OFFICE_OPEN_PLAN
+FAIL_MSG = "Lighting exists or is submitted with design documents. Lighting power density in P_RMR does not match U_RMR."
```

Comment:
We should be consistent whether to use a constant for the messages at the top of the file and then use the constant in the initializer, or to write out the messages in the initializer itself.

I don't have a strong preference other than to be consistent.